Update Oregon.md

emarcey opened the pull request:
Closes #1267 
Closes #1268 
Closes #1269
Closes #1270 
Closes #1271

aaaaaron-ship-it commented on reports/Oregon.md:
```diff
@@ -3475,10 +3475,69 @@ id: or-portland-213
+
+
+
+
```
Needs tag, Links heading, and more descriptive title.

emarcey replied:
Yeah this just got goofed up by the formatter

aaaaaron-ship-it commented on reports/Oregon.md:
```diff
@@ -3475,10 +3475,69 @@ id: or-portland-213
 
+### Police charge, tackle, and pepper spray protesters | August 1st
+
+Police gave directions to protesters to walk in a certain direction. Protesters followed directions. Police charged at protestors, shoving many to the ground. One protester on a bicycle is knocked over. The protester lands on top of the bike and has trouble getting up. Police pepper spray protesters as well.
```
Might be a good idea to make it explicit that there were several instances. Also I think some details could be added (see below)

aaaaaron-ship-it commented on reports/Oregon.md:
```diff
@@ -3475,10 +3475,69 @@ id: or-portland-213
+
+**Links**
+
+* https://twitter.com/hungrybowtie/status/1289791877223350273
```
It's really tough to see, but I think a cop maces a protester on the ground here. Might want to mention this if that's what's happening.

aaaaaron-ship-it commented on reports/Oregon.md:
```diff
@@ -3475,10 +3475,69 @@ id: or-portland-213
+* https://twitter.com/hungrybowtie/status/1289791877223350273
+* https://twitter.com/hungrybowtie/status/1289792999291985922
+* https://twitter.com/hungrybowtie/status/1289794603244072961
+* https://twitter.com/PDocumentarians/status/1289792025580023808
```
Should probably mention the woman who is shoved to the ground, forced to get up and then shoved repeatedly while she tells officers she can't see.

aaaaaron-ship-it commented on reports/Oregon.md:
```diff
@@ -3475,10 +3475,69 @@ id: or-portland-213
+* https://twitter.com/hungrybowtie/status/1289792999291985922
+* https://twitter.com/hungrybowtie/status/1289794603244072961
+* https://twitter.com/PDocumentarians/status/1289792025580023808
+* https://twitter.com/45thabsurdist/status/1289792288533577729
```
Might want to mention the person shoved to the ground who can't get up.

emarcey replied:
Yeah this is the "person who has trouble getting up" there are several angles of the same thing here

aaaaaron-ship-it replied:
Ah I didn't see.

emarcey commented:
@aaaaaron-ship-it addressed some of your comments. I don't think 225 is really "multiple incidents" tho. as far as I can tell it's just a bunch of angles of the same thing.
